### ISSUE TITLE
NoMethodError: undefined method `empty?' for nil:NilClass

### ISSUE THREAD

**Issue by gregkare** [5apps/liquor-cabinet#87]
https://app.getsentry.com/5apps/liquor-cabinet/issues/143871850/

```
NoMethodError: undefined method `empty?' for nil:NilClass
  remote_storage/swift.rb:27:in `authorize_request'
    server.halt 401, "Unauthorized" if token.empty?
  /var/www/liquor-cabinet/releases/3fafac0d2d322c13f1a864954cd831fb1361d4c9/liquor-cabinet.rb:82:in `block (2 levels) in <class:LiquorCabinet>'
    storage.authorize_request(@user, @directory, token, no_key) unless request.options?
  sinatra/base.rb:1611:in `call'
    proc { |a,p| unbound_method.bind(a).call }
  sinatra/base.rb:1611:in `block in compile!'
    proc { |a,p| unbound_method.bind(a).call }
  sinatra/base.rb:1015:in `[]'
    block ? block[self, values] : yield(self, values)
...
(30 additional frame(s) were not displayed)

NoMethodError: undefined method `empty?' for nil:NilClass
```

**Comment by skddc** [comment]
Could be from one of the countless timeouts to Redis.

**Comment by galfert** [comment]
This has been fixed by #88
